package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "floeproject.org",
+    "version": "1.0.0",
+    "description": "FLOE Project Website",
+    "scripts": {
+        "build": "eleventy",
+        "eleventy": "eleventy --serve",
+        "eleventyport": "eleventy --serve --port=9778",
```

Comment:
Why are there two scripts to start eleventy? Are both needed?

Also, to be consistent with other IDRC web projects, I would like to propose that `eleventyport` be replaced with `start` (see below).

For example, in our web projects we have:
* `clean` - deletes node_modules and output / build directories.
* `build` - deletes any existing (old) output files, and generates the output.
* `install` - installs dependencies
* `start` - runs clean, build, then runs the server that watches for changes in the output

Examples:
* https://github.com/inclusive-design/idrc/blob/dev/package.json
* https://github.com/platform-coop-toolkit/pinecone/blob/dev/package.json

Reply:
The default port at which eleventy runs is 8080. the eleventyport script is for running it at a different port i.e 9778.
I've made the required changes.